### ISSUE TITLE
meaningless and strange error messages for krun options

### ISSUE THREAD

**Issue by dlucanu** [kframework/k-legacy#924]
The command

```
$ kompile tests/ring.spec -cMNM "RING" -cTRGT "spec" -cMD "concrete" -cPRMS ".Valuation"
```

gives the following error:

```
[Error] Internal: Uncaught exception thrown of type ProvisionException.
Please file a bug report at https://github.com/kframework/k/issues
```

The error source is the missing "=" character between option name and its value. 
The correct command is

```
$ krun tests/ring.spec -cMNM="RING" -cTRGT="spec"  -cMD="generic" -cPRMS=".Valuation"
```

Some time ago the tool used to give a more meaningful message for this error.

Strange is that the rule with "=" does not work for all options. For instance, the following command gives the same error: 

```
$ krun tests/ring.spec -cMNM="RING" -cTRGT="spec"  -cMD="generic" -cPRMS=".Valuation" --output="none"
[Error] Internal: Uncaught exception thrown of type ProvisionException.
Please file a bug report at https://github.com/kframework/k/issues
```

If we remove the character "=" in the "--output" option, then it works fine:

```
$ krun tests/ring.spec -cMNM="RING" -cTRGT="spec"  -cMD="generic" -cPRMS=".Valuation" --output "none"
require "../data-structures.k"
require "../arch-spec/spec-instance.k"
require "../ip-syntax.k"
module RING
...
```

**Comment by dwightguth** [comment]
Should be fixed in #927

**Comment by dwightguth** [comment]
Please reopen if this is not fixed.
